### PR TITLE
[PyTorch] Add quantized::add_relu

### PR DESCRIPTION
**Pull request by zrphercule**
Since we are using this instead of quantized::add in cv model.
pytest tests/nodes/quantized_add_relu_test.py

### CODE REVIEW

File: torch_glow/src/PyTorchModelLoader.cpp [hunk 1]
```diff
@@ -488,6 +488,16 @@ struct QuantizedUnpackedConv2dInputs {
   };
 };
 
+/// Indexes of quantized::add inputs.
```
Comment (jackm321) [inline torch_glow/src/PyTorchModelLoader.cpp]: add_relu

File: torch_glow/tests/nodes/quantized_add_relu_test.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import torch
+
+from tests.utils import jitVsGlow
+
+
+def test_quantized_add_relu_zerooffset():
+    """Basic test of the PyTorch quantized::add Node on Glow with zero offset."""
```
Comment (jackm321) [inline torch_glow/tests/nodes/quantized_add_relu_test.py]: add_relu

File: torch_glow/tests/nodes/quantized_add_relu_test.py [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    x = torch.tensor([1, 2, 3, 4], dtype=torch.float32)
+    y = torch.tensor([5, 6, 7, 8], dtype=torch.float32)
+
+    jitVsGlow(test_f, x, y, expected_fused_ops={"quantized::add_relu",
+                                                "aten::quantize_per_tensor",
+                                                "aten::dequantize"})
+
+
+def test_quantized_add_relu():
+    """Basic test of the PyTorch quantized::add Node on Glow."""
```
Comment (jackm321) [inline torch_glow/tests/nodes/quantized_add_relu_test.py]: add_relu

Comment (zrphercule) [reply torch_glow/tests/nodes/quantized_add_relu_test.py]: lolololol

**Comment by zrphercule** [comment]
> LGTM
> Does quantized::add_relu broadcast like add?

no, quantized add & add_relu does not broadcast in current version.  https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/native/quantized/cpu/qadd.cpp#L29

**Pull request closed without merging**